apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/pagerules/PageRuleController.java

```diff
@@ -28,22 +31,26 @@ public class PageRuleController {
 
     private final PageRuleFinderService pageRuleFinderService;
 
-    public PageRuleController(PageRuleService pageRuleService, UserDetailsProvider userDetailsProvider,PageRuleFinderService pageRuleFinderService) {
+    private final PageRuleCreator pageRuleCreator;
+
+    public PageRuleController(PageRuleService pageRuleService, UserDetailsProvider userDetailsProvider,
+            PageRuleFinderService pageRuleFinderService,
+            PageRuleCreator pageRuleCreator) {
         this.userDetailsProvider = userDetailsProvider;
         this.pageRuleService = pageRuleService;
         this.pageRuleFinderService = pageRuleFinderService;
+        this.pageRuleCreator = pageRuleCreator;
     }
 
-    @PostMapping
+    @PostMapping("/create-business-rule")
```

Comment:
Issue: In REST, to add a new item you typically do a `POST` to the collection. in this case, `/rules`. So unless there is a conflicting path, we should revert the addition of `/create-business-rule` to the path.